ODP-Runtime.md

```diff
@@ -42,22 +43,22 @@ To help achieve TONE, ODP-compliant Hadoop platforms MUST conform to the followi
 Hadoop Version Specifications
 -----------------------------
 
--   ODP Platforms MUST be a descendent of the Apache Hadoop 2.7 branch.
+-   For this version of the specification, ODP Platforms MUST be a descendent of the Apache Hadoop 2.7 branch.  Future versions MAY increase the base Apache Hadoop version.
+
+-   The Apache components in an ODP reference release MUST have their source be 100% committed to an Apache source tree.
```

Comment:
The problem with Patch Available is that it means it might change or get dropped entirely.  It's *extremely* risky to take on something that hasn't been committed. It also means that an ODP spec *cannot* be recreated directly from the Apache source release, which to me at least is unacceptable.